.codecov.yml

```diff
@@ -0,0 +1,39 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+coverage:
+  precision: 4
+  round: down
+
+  status:
+    project:
+      default:
+        threshold: 3 #Allow the coverage to drop by threshold%, and posting a success status.
+    patch:
+      default:
+        target: 0%
+    changes: no
+
+parsers:
+  gcov:
+    branch_detection:
+      conditional: yes
+      loop: yes
+      method: no
+      macro: no
+
+comment:
+  layout: "header, diff"
+  behavior: default
+  require_changes: no
+
+ignore:
+  - "LICENSES"
+  - "*_test.go"
```

Comment:
whether needing to ignore ".pb.go" files?

Reply:
yep, can also add.
but we have already removed these files in build scripts

Reply:
added